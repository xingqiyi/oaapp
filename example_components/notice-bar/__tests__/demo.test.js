import { webDemoTest } from '../../../tests/shared/demoTest';

webDemoTest('notice-bar');
