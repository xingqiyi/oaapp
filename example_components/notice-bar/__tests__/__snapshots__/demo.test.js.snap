// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/notice-bar/demo/basic.md correctly 1`] = `
<div>
  <div
    class="am-whitespace am-whitespace-lg"
  />
  <div
    class="am-notice-bar"
    role="alert"
  >
    <div
      aria-hidden="true"
      class="am-notice-bar-icon"
    >
       
      <svg
        class="am-icon am-icon-trips am-icon-xxs"
      >
        <use
          xlink:href="#trips"
        />
      </svg>
       
    </div>
    <div
      class="am-notice-bar-content"
    >
      <div
        class="am-notice-bar-marquee-wrap "
        role="marquee"
        style="overflow:hidden;"
      >
        <div
          class="am-notice-bar-marquee"
          style="position:relative;right:0;white-space:nowrap;display:inline-block;padding:0 0.15rem;"
        >
          Notice: The arrival time of incomes and transfers of Yu 'E Bao will be delayed during National Day. 
        </div>
      </div>
    </div>
  </div>
  <div
    class="am-whitespace am-whitespace-lg"
  />
  <div
    class="am-notice-bar"
    role="alert"
  >
    <div
      aria-hidden="true"
      class="am-notice-bar-icon"
    >
       
      <svg
        class="am-icon am-icon-trips am-icon-xxs"
      >
        <use
          xlink:href="#trips"
        />
      </svg>
       
    </div>
    <div
      class="am-notice-bar-content"
    >
      <div
        class="am-notice-bar-marquee-wrap "
        role="marquee"
        style="overflow:hidden;"
      >
        <div
          class="am-notice-bar-marquee"
          style="position:relative;right:0;white-space:nowrap;display:inline-block;"
        >
          Notice: The arrival time of incomes and transfers of Yu 'E Bao will be delayed during National Day. 
        </div>
      </div>
    </div>
    <div
      aria-label="go to detail"
      class="am-notice-bar-operation"
      role="button"
    >
      <svg
        class="am-icon am-icon-right am-icon-md"
      >
        <use
          xlink:href="#right"
        />
      </svg>
    </div>
  </div>
  <div
    class="am-whitespace am-whitespace-lg"
  />
  <div
    class="am-notice-bar"
    role="alert"
  >
    <div
      class="am-notice-bar-content"
    >
      <div
        class="am-notice-bar-marquee-wrap "
        role="marquee"
        style="overflow:hidden;"
      >
        <div
          class="am-notice-bar-marquee"
          style="position:relative;right:0;white-space:nowrap;display:inline-block;"
        >
          Remove the default icon. 
        </div>
      </div>
    </div>
    <div
      aria-label="close"
      class="am-notice-bar-operation"
      role="button"
    >
      <svg
        class="am-icon am-icon-cross am-icon-md"
      >
        <use
          xlink:href="#cross"
        />
      </svg>
    </div>
  </div>
  <div
    class="am-whitespace am-whitespace-lg"
  />
  <div
    class="am-notice-bar"
    role="alert"
  >
    <div
      aria-hidden="true"
      class="am-notice-bar-icon"
    >
       
      <svg
        class="am-icon am-icon-check-circle-o am-icon-xxs"
      >
        <use
          xlink:href="#check-circle-o"
        />
      </svg>
       
    </div>
    <div
      class="am-notice-bar-content"
    >
      <div
        class="am-notice-bar-marquee-wrap "
        role="marquee"
        style="overflow:hidden;"
      >
        <div
          class="am-notice-bar-marquee"
          style="position:relative;right:0;white-space:nowrap;display:inline-block;"
        >
          Customized icon. 
        </div>
      </div>
    </div>
    <div
      aria-label="close"
      class="am-notice-bar-operation"
      role="button"
    >
      <svg
        class="am-icon am-icon-cross am-icon-md"
      >
        <use
          xlink:href="#cross"
        />
      </svg>
    </div>
  </div>
</div>
`;
